orchestrator-salt/src/main/resources/salt/salt/post-recipes/scripts/createuserhome.sh

```diff
@@ -38,6 +38,8 @@ EXISTING_USERS_FILE=/tmp/existing_users_file
 HDFS_KEYTAB=/run/cloudera-scm-agent/process/$(ls -t /run/cloudera-scm-agent/process/ | grep hdfs-NAMENODE$ | head -n 1)/hdfs.keytab
 echo "Keytab dir:$HDFS_KEYTAB"
 
+PATH_PREFIX="/user7"
```

Comment:
Do we need this prefix?

Reply:
Oh, the prefix should be set to /user .. I had set to user7 for testing and forgot to update.